### ISSUE TITLE
 ​ Any chance to have something like ant.design for file uploads in $f7?

### ISSUE THREAD

**Issue by rapgithub** [framework7io/framework7#4128]
ant.design has a nice component for image upload and it will be cool f7 will incorporate it because there is no upload component yet inside f7 and it will be nice this kind of interface to upload avatars for profiles or just using it for an app where image management and upload is needed!

here the link:
https://ant.design/components/upload

npm link:
https://www.npmjs.com/package/antd

**Comment by Simone4e** [comment]
Seems like a good improvement UI for the future

**Comment by rapgithub** [comment]
Yes it is! indeed! I hope you can consider it as new component for f7 pretty soon! :)

**Comment by nolimits4web** [comment]
This issue was incorrectly closed on 2026-05-19 by a compromised credential (Shai-Hulud npm supply-chain worm). Reopening — apologies for the noise.
